app/helpers/events_helper.rb

```diff
@@ -25,6 +25,10 @@ def event_status_open?(event)
     event.status_id == GetIntoTeachingApiClient::Constants::EVENT_STATUS["Open"]
   end
 
+  def event_status_pending?(event)
```

Comment:
I don't think this is used?

Reply:
I forgot a change here: https://github.com/DFE-Digital/get-into-teaching-app/pull/1118/commits/cd92bc621ec3129697242c9ceb27201bbbd4faec

It didn't look great having the banner showing on event pending event